java/api/src/main/java/org/ray/api/options/ActorCreationOptions.java

```diff
@@ -10,26 +10,31 @@
 
   public static final int NO_RECONSTRUCTION = 0;
   public static final int INFINITE_RECONSTRUCTIONS = (int) Math.pow(2, 30);
+  public static final boolean DEFAULT_IS_DIRECT_CALL = "1"
+      .equals(System.getenv("ACTOR_CREATION_OPTIONS_DEFAULT_IS_DIRECT_CALL"));
 
   public final int maxReconstructions;
 
+  public final boolean isDirectCall;
+
   public final String jvmOptions;
 
-  private ActorCreationOptions(Map<String, Double> resources,
-                               int maxReconstructions,
-                               String jvmOptions) {
+  private ActorCreationOptions(Map<String, Double> resources, int maxReconstructions,
+      boolean isDirectCall, String jvmOptions) {
     super(resources);
     this.maxReconstructions = maxReconstructions;
+    this.isDirectCall = isDirectCall;
```

Comment:
I prefer naming it `useDirectCall`. what do you think?

Reply:
FYI, you can find 40+ results if you search for `is_direct_call` in the codebase.